src/scripts/components/share/share.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {FunctionComponent, useState} from 'react';
+
+import {YoutubeIcon} from '../icons/youtube-icon';
+import {TwitterIcon} from '../icons/twitter-icon';
+import {InstagramIcon} from '../icons/instagram-icon';
+import {FacebookIcon} from '../icons/facebook-icon';
+import {CopyIcon} from '../icons/copy-icon';
+import {ShareIcon} from '../icons/share-icon';
+import config from '../../config/main';
+
+import styles from './share.styl';
+import {FormattedMessage} from 'react-intl';
+import Button from '../button/button';
+import {CloseIcon} from '../icons/close-icon';
+import Overlay from '../overlay/overlay';
+import {replaceUrlPlaceholders} from '../../libs/replace-url-placeholders';
+
+const Share: FunctionComponent = () => {
+  const [showShare, setShowShare] = useState(false);
+  const currentUrl = window.location.href;
+  const shareUrl = `ESA Climate From Space: ${window.location.href}`;
+  const facebookUrl = replaceUrlPlaceholders(config.api.facebook, {
+    currentUrl
+  });
+  const twitterUrl = replaceUrlPlaceholders(config.api.twitter, {
+    currentUrl
+  });
+  const urlInput = document.querySelector('#urlInput') as HTMLInputElement;
+  const copyUrl = () => {
+    if (navigator.clipboard) {
+      navigator.clipboard.writeText(shareUrl);
+    } else {
+      urlInput.select();
+      document.execCommand('copy');
+    }
+  };
+
+  return (
+    <div className={styles.share}>
+      <Button
+        className={styles.shareButton}
+        icon={ShareIcon}
+        onClick={() => setShowShare(true)}
+      />
+      {showShare && (
+        <Overlay showCloseButton={false} onClose={() => {}}>
```

Comment:
Why is onClose an empty function? Better mark it optional and give it an empty function as its default value